openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1500,9 +1500,36 @@ Credential Refresh can be initiated by the Wallet independently from the Credent
 
 It is up to the Credential Issuer whether to update both the signature and the claim values, or only the signature.
 
-# Privacy Considerations
+# Privacy Considerations {#privacy-considerations}
 
-TBD
+Implementers and deployers should utilize mechanisms to protect End-User privacy.
+Means of protecting End-User privacy are described below.
+
+Privacy harms can occur if information about a person is released to another party without the person's consent.
+True consent involves both the person knowing what information is being released and knowing who it is being released to, and for what purpose.
+The means to ensure the session's integrity to prevent disclosure to unintended parties, including Wallets not under control of the End-User, are means to protect privacy.
+
+Privacy-preserving mechanisms such as selective disclosure should be used to protect End-User privacy, when possible.
+If supported by the Credential format, the Issuer should make choices about which claims are selectively disclosable.
+Claims that are always disclosed can enable correlation by verifiers.
+Another possibility is issuing Credentials containing only a single claim each.
+
+Furthermore, multiple seemingly innocent consenting information releases can result in privacy erosion through collusion and correlation.
+There may be unintended second or third order effects, which is why minimal disclosure is crucial.
+
+Multiple Credentials can be linkable via claim data, identifiers, and other credential metadata.
+Batch issuance can solve this potential privacy issue, enabling each issued credential to only be used once.
```

Comment:
I think this sentence probably overstates things a little as batch issuance can only help with one small part of the problem the previous sentence introduces - it does not help at all if the claim data or identifiers are linkable.

Here's a suggestion though it feels a little clumsy:

```suggestion
Using an issued credential only once can help avoid some of these concerns for some credential formats.  Batch issuance can help achieve the wallet obtain copies of a credential.
```

Reply:
@jogu can you change "Batch issuance can help achieve the wallet obtain copies of a credential." in your suggestion to "Batch issuance can be used by the wallet obtain multiple copies of a credential."  The current suggestion text is confusing.

Reply:
I think confusing was polite, I'd clearly got halfway through rewriting the sentence and seemingly given up 😆

I've updated my suggestion with your wording (except I added a missing 'to' before 'obtain').